inim.nimble

```diff
@@ -9,7 +9,7 @@ bin           = @["inim"]
 
 # Dependencies
 
-requires "nim >= 1.0.0"
+#requires "nim >= 1.0.0" # can we remove this to imply it should work with all versions?
```

Comment:
is inim working with nim < 1.0.0 ? if not then you should keep this

Reply:
@timotheecour thanks Tim, but the test fails on devel with this line, how can I get round this?

Reply:
Here's the error using `nimlang/nim:devel` and the require nim 1 line:
```
$ nimble build
Unsatisfied dependency: nim (>= 1.0.0)
```

Reply:
you may need to track down which version of nim is installed that leads to this failure.
try:

run: nimble build
=>
run: nim -v && nimble build --verbose

or something similar to get more debug info

Reply:
Much obliged @timotheecour , trying now.

Reply:
`Nim Compiler Version 0.15.3 (2016-12-13) [Linux: amd64]`

Fun! I wonder why the docker container isn't using the latest. Must be the wrong tag, I'll see what the right one is.